rero_ils/modules/loans/api.py

```diff
@@ -560,7 +560,7 @@ def _process_patron_profile_fees(patron, organisation, status='open'):
         if 'loan' in transaction:
             item_pid = Loan.get_record_by_pid(transaction.loan.pid)\
                 .get('item_pid')
-            item = Item.get_record_by_pid(item_pid)
+            item = Item.get_record_by_pid(item_pid.get('value'))
```

Comment:
There is something wrong elsewhere if you have to use `item_pid.get('value')`.
`item_pid` should be already a pid value and not a dictionary.
In the loan json schema this is declared as string value!

Reply:
This comes form invenio-circulation

Reply:
https://github.com/inveniosoftware/invenio-circulation/blob/c3fcc1a4674d5df24221e3c068e8e0c07db68016/invenio_circulation/mappings/v5/loans/loan-v1.0.0.json#L39

Reply:
`item_pid` is a loan in this context. And the loan jsonschema have properties type and value here: https://github.com/rero/rero-ils/blob/US1394-invenio-circulation/rero_ils/modules/loans/mappings/v6/loans/loan-ils-v0.0.1.json#L51